src/lib/dialog/dialog-container.ts

```diff
@@ -101,6 +101,7 @@ export class MatDialogContainer extends BasePortalOutlet {
     public _config: MatDialogConfig) {
 
     super();
+    this._ariaLabelledBy = _config.ariaLabelledBy || null;
```

Comment:
This should allow us to remove the initializer from the `_ariaLabelledBy` property.